mindsdb/integrations/handlers/gcs_handler/requirements.txt

```diff
@@ -0,0 +1,5 @@
+duckdb
+fsspec
+gcsfs
+google-auth
+google-cloud-storage
```

Comment:
In order for the use of these packages to be accepted by our CI checks, you will need to add them to this [dict](https://github.com/mindsdb/mindsdb/blob/da0e83985974d7796897c25b3c40ecf57c17e485/tests/scripts/check_requirements.py#L71). I think looking at the examples should allow you to understand how these are handled, but if you have any questions, let me know.

Reply:
Done